CONTRIBUTION.md

```diff
@@ -0,0 +1,117 @@
+# CONTRIBUTION GUIDELINES
+Welcome to the Hyperledger Labs AIFAQ project! We're excited to have you contribute and help us improve our project. Below are the guidelines to ensure a smooth and efficient contribution process.
+
+## Code of Conduct
+Please adhere to our [Code of Conduct](https://wiki.hyperledger.org/pages/viewpage.action?pageId=41587043). This code of conduct outlines our expectations for participants within the community as well as steps to report unacceptable behavior.
+
+## Getting Started
+
+### Development Server
```

Comment:
Split into Frontend and Backend development work and their relevant steps.

Reply:
Splitted into frontend and backend